src/constants/index.tsx

```diff
@@ -528,3 +528,18 @@ export enum Theme {
   Dark = "dark",
   Gray = "gray",
 }
+
+export const PageTitle = {
+  ABOUT: `About`,
+  ENTERPRISE: `Enterprise`,
+  CONTACT: `Contact`,
+  PLAYGROUND: `Playground`,
+}
```

Comment:
The final title should be specified over here. Instead of creating the string dynamically where it's used. So instead of `About` make it `About | ${tagline}`.

Reply:
```ts
export const PageTitle = {
  ABOUT: `About | ${tagline}`,
  ENTERPRISE: `Enterprise | ${tagline}`,
  CONTACT: `Contact | ${tagline}`,
  PLAYGROUND: `Playground | ${tagline}`,
}
```

Something like this should work.